src/CatBoost.jl

```diff
@@ -135,24 +110,14 @@ to_catboost(arg) = Tables.istable(arg) ? to_pandas(arg) : arg
 # utility for calling `to_catboost` on each argument of a function
 all_to_catboost(args) = (to_catboost(arg) for arg in args)
 
-# the Julia-side code does not copy the columns, but the `pandas.DataFrame`
-# constructor seems to make a copy here. Maybe that can be avoided?
 function to_pandas(tbl)
-    # ensure we have a column table
-    col_table = Tables.columns(tbl)
-    # write it in a way that pandas will understand (after PyCall conversion)
-    dict_table = OrderedDict(col => Tables.getcolumn(col_table, col)
-                             for col in Tables.columnnames(col_table))
-    return pandas.DataFrame(; data=dict_table)
+    pandas_df = pandas.DataFrame(eachcol(tbl)).T
+    pandas_df.columns = names(tbl)
+    return pandas_df
 end
 
-function pandas_to_df(pandas_df::PyObject)
-    colnames = map(pandas_df.columns) do c
-        ret = c isa PyObject ? PyAny(c) : c
-        return ret isa Int ? ret + 1 : ret
-    end
-    df = DataFrame(Any[Array(getproperty(pandas_df, c).values) for c in colnames],
-                   map(Symbol, colnames))
+function pandas_to_df(pandas_df::Py)
+    df = DataFrame(PyPandasDataFrame(pandas_df))
```

Comment:
I think this looks good, but from the [docs](https://cjdoris.github.io/PythonCall.jl/stable/compat/#Tabular-data-and-Pandas) it sounds like we can also go with:
```suggestion
    df = DataFrame(PyTable(pandas_df))
```